repository-data/webfiles/src/main/resources/site/apicatalogue/apicatalogue.js

```diff
@@ -67,19 +67,20 @@ function findHTMLElementData(result, element, htmlElement) {
 function replaceHTMLContent(result, element, text, searchTerm, htmlElement) {
     const highlightContent1 = text.replaceAll('<span class="filter-tag-yellow-highlight">', '');
     const highlightContent2 = highlightContent1.replaceAll('</span>', '');
+    const highlightContent3 = highlightContent2.replaceAll('&amp;', '&');
```

Comment:
These three `replaceAll` statements can be combined to save on intermediate variables. 
```
const highlightContent = text
    .replaceAll('<span class="filter-tag-yellow-highlight">', '')
    .replaceAll('</span>', '')
    .replaceAll('&amp;', '&');
```
By chaining these calls together you improve clarity.

Reply:
This is amended as above. Thank you for your comments and approval @GeoffreyHayward.

Reply:
No problem.